flows/expressions.go

```diff
@@ -38,8 +38,10 @@ var RunContextTopLevels = []string{
 	"globals",
 	"input",
 	"legacy_extra",
+	"node",
```

Comment:
this is only needed for parsing of expressions without a real context - i.e. possible future flow migrations